tests/models/test_tpu.py

```diff
@@ -51,42 +51,24 @@ def test_model_tpu_cores_1(tmpdir):
     tpipes.run_model_test(trainer_options, model, on_gpu=False, with_hpc=False)
 
 
+@pytest.mark.parametrize('tpu_core', [1, 5])
```

Comment:
I thought this did not work (test skipped) and the test had to be split on purpose? Are you sure about it

Reply:
It did not work with spawn decorator... If you open the TPU test details you should see all tests passed...

Reply:
In fact it was failing for a while because of bad assets =)